packages/utils/src/misc.ts

```diff
@@ -21,7 +21,7 @@ interface SentryGlobal {
 }
 
 /**
- * Requires a module which is protected _against bundler minification.
+ * Requires a module which is protected against bundler minification.
```

Comment:
This is what got me started... I thought "why __against_?!". The rest is Git history.